lib/CrudService.js

```diff
@@ -381,11 +381,11 @@ class CrudService {
       operations.push(operation)
     }
 
-    const { ok, nModified, nUpserted } = await this._mongoCollection.bulkWrite(operations)
+    const { ok, modifiedCount, nUpserted } = await this._mongoCollection.bulkWrite(operations)
```

Comment:
Should we also modify all the instances of `nUpserted` in `BulkWriteResult` to [`upsertedCount`](https://mongodb.github.io/node-mongodb-native/6.1/classes/BulkWriteResult.html#upsertedCount)?